.probo.yaml

```diff
@@ -1,78 +1,21 @@
-      echo 'export PATH="/usr/bin/mysql:$PATH"' >> /root/.bash_profile
-      source /root/.bash_profile
-      mysql -u root -e "SET PASSWORD FOR 'root'@'localhost' = PASSWORD('root')";
-      mysql -u root -proot -e "CREATE DATABASE html";
-      # Downloading Probo CI Assets
-      git clone https://github.com/SU-SWS/proboci_assets.git /srv/proboci_assets
-      cp /srv/proboci_assets/.my.cnf /root/.my.cnf
-      # Installing drush 8
-      sudo mkdir --parents /opt/drush-8.x
-      cd /opt/drush-8.x
-      sudo composer init --require=drush/drush:8.* -n
-      sudo composer config bin-dir /usr/local/bin
-      sudo composer install &>/dev/null &
-      # Installing node modules
-      npm install -g npm >/dev/null
-      npm install -g selenium-standalone chromedriver >/dev/null
-      selenium-standalone install --silent
-      # Probo CI fails to start selenium-standalone start when run with their command plugin.
-      # But we might need to start selenium in the same shell as we run Behat tests.
-      # And force Probo CI to display Behat test results, if using their script plugin.
-      nohup selenium-standalone start &>/dev/null &
-      # Installing Behat
-      git clone https://github.com/SU-SWS/linky_clicky.git /srv/linky_clicky
-      cd /srv/linky_clicky
-      composer install &>/dev/null &
-      cp /srv/proboci_assets/behat.local.yml /srv/linky_clicky/sites/probo/behat.local.yml
-      cp /srv/linky_clicky/includes/features/SU-SWS/$test_feature/$test_feature.feature /srv/linky_clicky/sites/probo/features/.
-      # Downloading and running Jumpstart Deployer
-      git clone git@github.com:SU-SWS/stanford-jumpstart-deployer.git /srv/stanford-jumpstart-deployer
-      cd /srv/stanford-jumpstart-deployer
-      drush make development/product/$test_deployer_product/$test_deployer_product.make /var/www/html
-      cp /srv/proboci_assets/.htaccess /var/www/html/.htaccess
-      mkdir -p /var/www/html/sites/default/files/styles
-      chmod -R 777 /var/www/html/sites/default/files
-  - name: Building site
-    command: '
-      df -h;
-      cd /var/www/html/profiles;
-      test_product_profile=$(echo stanford_sites_jumpstart_*);
-      service mysql start;
-      service mysql reload;
-      service mysql restart;
-      cd /var/www/html;
-      drush si $test_product_profile --account-name=admin --db-url="mysql://root:root@localhost/html"'
-  - name: Run relevant Behat tests
-    command: '
-      test_feature=$(cd /src | echo *.info | cut -f1 -d".");
-      cd /srv/linky_clicky/sites/probo;
-      /srv/linky_clicky/bin/behat features/$test_feature.feature --profile default --suite dev --format pretty'
+  - name: Run tests from external proboci_assets repository
+    command: "git clone https://github.com/SU-SWS/proboci_assets.git /srv/proboci_assets;
+      cp $ASSET_DIR/proboci_id_rsa.pub /root/.ssh/id_rsa.pub;
+      cp $ASSET_DIR/proboci_id_rsa /root/.ssh/id_rsa;
+      /srv/proboci_assets/probo.sh stanford"
```

Comment:
Is it possible to just call the script and not have 'stanford' at the end? The reason for this being is that if we want to switch the default profile for all of our repos later it would be nice to do that in the assets repo instead of in each feature repo. Please leave alternative examples as comments.

eg: # /srv/proboci_assets/probo.sh stanford
      # /srv/proboci_assets/probo.sh jumsptart